spm_funcs.py

```diff
@@ -54,8 +54,16 @@ def get_spm_globals(fname):
     spm_vals : array
         SPM global metric for each 3D volume in the 4D image.
     """
-    # +++your code here+++
-    # return
+    #- Load the image given by "fname".
+    img = nib.load(fname)
+    #- Get the data
+    data = img.get_fdata()
+    #- Calculate the SPM global value for each volume.
+    spm_vals = []
+    for vol in range(data.shape[-1]):
+        glob = spm_global(data[..., vol])
+        spm_vals.append(glob)
+    return spm_vals  # Return the result.
```

Comment:
What is the type of `spm_vals`?

If you read the docstring (line 54 above) it says `array`, which is very unclear but seems to refer to a `numpy.array`, however, you return a basic `list` here. Consider converting it to array before returning or else downstream methods could fail. For instance, with the current implementation you would not be able to do:
```Python
average_spm = get_spm_globals(nifti_file).mean()
```